promptsource/templates/GEM/mlsum/de/templates.yaml

```diff
@@ -0,0 +1,86 @@
+dataset: GEM/mlsum
+subset: de
+templates:
+  104b2645-9b2a-4d08-8885-3de18af246fc: !Template
+    answer_choices: null
+    id: 104b2645-9b2a-4d08-8885-3de18af246fc
+    jinja: "My college roommate asked me what this German article meant:\n {{text}}\n\
+      So I recapped it in layman''s terms in German: ||| {{target}}"
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - ROUGE
+      - BLEU
+      original_task: true
+    name: layman_summ_de
+    reference: http://gptprompts.wikidot.com/prompt:summarization
+  6ffadb8a-c670-4d6c-97fd-9eea35945452: !Template
+    answer_choices: null
+    id: 6ffadb8a-c670-4d6c-97fd-9eea35945452
+    jinja: "{{text}}\n TL;DR in German: ||| {{target}}"
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - ROUGE
+      - BLEU
+      original_task: true
+    name: tl_dr_de
+    reference: ''
+  a4f6a6c1-ce10-463b-932e-41a9336c3ecf: !Template
+    answer_choices: null
+    id: a4f6a6c1-ce10-463b-932e-41a9336c3ecf
+    jinja: "{{text}}\n      ===\nWrite a summary of the text above : ||| {{target}}"
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - ROUGE
+      - BLEU
+      original_task: true
+    name: write_summary_of_above
+    reference: ''
+  e071fb9c-b000-417d-a405-4f6032532f87: !Template
+    answer_choices: null
+    id: e071fb9c-b000-417d-a405-4f6032532f87
+    jinja: "{{text}}\n      ===\nGiven the above document, write one sentence in german\
```

Comment:
MLSum has multiple sentences so the "write one sentence" wouldn't work here.

Reply:
Fixed